include/ensmallen_bits/cmaes/cmaes_impl.hpp

```diff
@@ -99,13 +105,16 @@ typename MatType::elem_type CMAES<SelectionPolicyType>::Optimize(
 
   std::vector<BaseMatType> mPosition(2, BaseMatType(iterate.n_rows,
       iterate.n_cols));
-  mPosition[0] = lowerBound + arma::randu<BaseMatType>(
-      iterate.n_rows, iterate.n_cols) * (upperBound - lowerBound);
+  BaseMatType initialVal;
+  initialVal.randu(iterate.n_rows, iterate.n_cols);
+  initialVal += (BaseMatType)(iterateIn);
+  mPosition[0] = initialVal;
```

Comment:
I think it's still valid to apply the lower/upper bound here.

Reply:
Hi, @zoq so should I remove the boundary transformation from [here](https://github.com/mlpack/ensmallen/pull/193/files#diff-98ddaa22db898dc556617924f1ad158ab9603064325059c236d604daeba55b56R117) and implement it like it was before like [here](https://github.com/mlpack/ensmallen/blob/master/include/ensmallen_bits/cmaes/cmaes_impl.hpp#L102). Thanks